g7_declaration/SQ1-1e.yml

```diff
@@ -1,6 +1,5 @@
 question: What is your organisation's company registration number?
 hint: >
   If you don't have a Companies House number, provide the company registration
-  number from the country where you first registered. If you don't have an
-  organisation, you don't have to answer this question.
```

Comment:
It’s not optional anymore. Cath has written a story to fix the validation: https://www.pivotaltracker.com/story/show/102066014
On 26 Aug 2015, at 14:45, Kevin Keenoy <notifications@github.com> wrote:

> In g7_declaration/SQ1-1e.yml:
> 
> > @@ -1,6 +1,5 @@
> >  question: What is your organisation's company registration number?
> >  hint: >
> >    If you don't have a Companies House number, provide the company registration
> > -  number from the country where you first registered. If you don't have an
> > -  organisation, you don't have to answer this question.
> etc.
> 
> —
> Reply to this email directly or view it on GitHub.
>